src/d/actor/d_a_lbridge.cpp

```diff
@@ -6,99 +6,327 @@
 #include "d/actor/d_a_lbridge.h"
 #include "d/d_procname.h"
 
+const char daLbridge_c::m_arcname[] = "Gbrg00";
+
 /* 00000078-00000098       .text CheckCreateHeap__FP10fopAc_ac_c */
-static BOOL CheckCreateHeap(fopAc_ac_c*) {
-    /* Nonmatching */
+static BOOL CheckCreateHeap(fopAc_ac_c* i_this) {
+    return static_cast<daLbridge_c*>(i_this)->CreateHeap();
 }
 
 /* 00000098-00000384       .text CreateHeap__11daLbridge_cFv */
-void daLbridge_c::CreateHeap() {
-    /* Nonmatching */
+BOOL daLbridge_c::CreateHeap() {
+    J3DModelData* modelData = (J3DModelData*)dComIfG_getObjectRes(m_arcname, 7);
```

Comment:
FYI, last week I generated enums for all the arc files in the game, so now you can include the one for this arcname like:
```
#include "d/res/res_gbrg00.h"
```
And then these getObjectRes calls can look like this with the filenames instead of numbers:
```suggestion
    J3DModelData* modelData = (J3DModelData*)dComIfG_getObjectRes(m_arcname, GBRG00_BDL_GBRG00);
```
This can help make the code more readable for some actors.